third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -166,6 +164,10 @@ impl Loader {
         })
     }
 
+    pub(crate) fn v2(natives: NativeFunctions, vm_config: VMConfig) -> Self {
+        Self::V2(LoaderV2::new(natives, vm_config, DummyVerifier))
```

Comment:
I think DefaultVerifier is a better name? It's actually not dummy because it runs the basic checks needed.

Reply:
It used to be dummy, made not dummy to see how many tests pass😄 cann roll back this change, I would prefer to keep is a dummy because it should go away :)

Reply:
@runtian-zhou removed it! It is actually better now, we take `Option<dyn VerifierExtension>` to add checks beyond bytecode verifier